docs/recipes/debugging-with-webstorm.md

```diff
@@ -13,6 +13,8 @@ In the `JavaScript file` field specify the path to AVA in the project's `node_mo
 
 In the `Application parameters` pass the CLI flags you're using and the test files you would like to debug, for example `--verbose test.js`.
 
+In the `Node parameters` pass the `--inspect-brk` flag to enable the Node inspector or `--inspect-brk=port` if you want to listen it on a specific port.
```

Comment:
@melisoner2006 Please add note that `--inspect-brk` works only in the node 7+.

Not all users uses latest 8 :)